frontend/src/app/pages/map/Map.tsx

```diff
@@ -0,0 +1,17 @@
+import MapContainer from '../../components/Map/MapContainer';
+
+export default function Map() {
```

Comment:
Not required, but I think we gonna use arrow function so maybe change for the convention
```
const Map = () => {
  ...
}

export default Map;
```